com.tracktion.Waveform13.appdata.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.tracktion.Waveform13</id>
+  <name>Waveform 13</name>
+  <icon type="stock">waveform13</icon>
```

Comment:
```suggestion
```

Reply:
I already tried that beforehand. The build always fails with the following error:
```
Composing metadata...
Run failed, some data was ignored.
Errors were raised during this compose run:
com.tracktion.Waveform13
  E: metainfo-parsing-error

general
  E: filters-but-no-output
Refer to the generated issue report data for details on the individual problems.
Error: ERROR: appstreamcli compose failed: Child process exited with code 1```

Reply:
No remove that first

Reply:
it's because you don't install a .desktop file. So appstream compose doesn't find the icon to resolve from there. And there will be no icon found with that stock name anyway.